eng/pipelines/aggregate-reports.yml

```diff
@@ -32,11 +32,29 @@ jobs:
           & "tools/AnalyzeDeps/AnalyzeDeps.ps1" -PackagesPath "$(System.ArtifactsDirectory)/BuildArtifacts/*.nupkg" -LockfilePath "$(Build.SourcesDirectory)/eng/Packages.Data.props" `
           -OutPath "$(Build.ArtifactStagingDirectory)/reports/dependencies.html" -DumpPath "$(Build.ArtifactStagingDirectory)/reports/data.js"
         displayName: "Analyze dependencies"
+
+      - task: UseDotNet@2
+        displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
+        inputs:
+          version: "$(DotNetCoreSDKVersion)"
+
+      - task: UseDotNet@2
```

Comment:
Since we aren't actually running an code we don't need to set the runtime, setting only the sdk should be all we need.